driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/syncadapter/SyncMongoDatabase.java

```diff
@@ -206,12 +206,16 @@ public void createCollection(final String collectionName) {
 
     @Override
     public void createCollection(final String collectionName, final CreateCollectionOptions createCollectionOptions) {
-        throw new UnsupportedOperationException();
+        SingleResultSubscriber<Void> subscriber = new SingleResultSubscriber<>();
+        wrapped.createCollection(collectionName).subscribe(subscriber);
```

Comment:
Should be

    wrapped.createCollection(collectionName, createCollectionOptions)...